Makefile

```
@@ -28,26 +28,15 @@ endif
 RPM_BUILD_OPTIONS := --with=mysql
 
 OSUSE_REPOS = https://download.opensuse.org/repositories
-ifneq ($(REPOSITORY_URL),"")
-ifneq ($(DAOS_STACK_SLES_12_GROUP_REPO),"")
-sle12_REPOS += --repo $(REPOSITORY_URL)$(DAOS_STACK_SLES_12_GROUP_REPO)
-endif
-ifneq ($(DAOS_STACK_SLES_12_LOCAL_REPO),"")
-sle12_REPOS += --repo $(REPOSITORY_URL)$(DAOS_STACK_SLES_12_LOCAL_REPO)
-endif
-ifneq ($(DAOS_STACK_LEAP_42_GROUP_REPO),"")
-sl42_REPOS += --repo $(REPOSITORY_URL)$(DAOS_STACK_LEAP_42_GROUP_REPO)
-endif
-ifneq ($(DAOS_STACK_LEAP_42_LOCAL_REPO),"")
-sl42_REPOS += --repo $(REPOSITORY_URL)$(DAOS_STACK_LEAP_42_LOCAL_REPO)
-endif
-else
+ifeq ($(REPOSITORY_URL),"")
```

```suggestion
ifeq ($(REPOSITORY_URL),)
```

I think is the correct/idiomatic way/  `""` might not actually be the same thing as "nothing" to `make`.

---

It is not.  I missed fixing this earlier.